src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1233,6 +1266,39 @@ private void UpdateEditor(float delta)
         {
             CurrentOrganelleCost = 0;
         }
+
+        var vector = Vector3.Zero;
+        if (cameraMovementLeft)
+            vector += Vector3.Left;
+        if (cameraMovementRight)
+            vector += Vector3.Right;
+        if (cameraMovementUp)
+            vector += Vector3.Forward;
+        if (cameraMovementDown)
+            vector += Vector3.Back;
+
+        // Apply camera movement
+        camera.ObjectToFollow.Translation += vector.Normalized() * camera.CameraHeight * delta;
+    }
+
+    private void HandleMovement()
+    {
+        if (mousePanningStart == null)
```

Comment:
Found a bug regarding this, if you hold down a camera pan key and start mouse camera pan, the input key gets stuck on. So if mouse panning is happening all the other movement should be set to false.